### DISCUSSION TITLE
Import VM and rename

### DISCUSSION THREAD

**Discussion by netbja** [bb-Ricardo/netbox-sync#147] [Ideas]
Hello Ricardo,

A new idea to import VMs from vCenter by changing the name.

The fitst idea is that my VMs have a display name with a comment in brackets 
ex: SERVER1 (LDAP Server) 

It would be very useful to do the import by deleting the data between parenthesis and put the display name in lower case.
So I could use the dynamic inventory (ansible) on the VM hostname with --limit 

Second idea, an interesting trick would be to set custom fields with the vCenter import.
"vm_uuid": 56XXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX

Thank you for this great sync tool !

Bernard.
--

**Comment by bb-Ricardo** [comment]
Hi Bernard,

Your 1st idea:
I don't really get what you are trying to do. Do you mean `SERVER1 (LDAP Server)` should become `server1`?

This would be quite difficult to put into config as there are so many possibilities of how to change a name.

Your 2nd idea:
This sounds quite interesting and quite easy to add. This could be a config option with a comma separated list of VM attributes which then get added as Custom attribute to each VM.

**Comment by netbja** [comment]
HI Ricardo,

Yes, for my first idea SERVER1 (LDAP Server) -> server1, I understand that it's a bit of a crazy idea...

But don't worry, I'm going to creae a script to rename the VMs names.

For the second one, as soon as it is ready, I can test for you  :-)
